### PR TITLE
docs(platform/codecommit): add aws codebuild examples

### PR DESCRIPTION
**Pull request by PhilipAbed**
## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/platform/codecommit/index.md [hunk 1]
```diff
@@ -143,3 +143,56 @@ module.exports = {
   ],
 };
 ```
+
+## CodeBuild examples
+
+Create a repository with buildspec.yml.
+This repository will be your BuildProject job repository to run renovate on your repositories
```
Comment (HonkingGoose) [inline lib/modules/platform/codecommit/index.md:150]:
```suggestion
This repository will be your BuildProject job repository to run Renovate on your repositories.
```

File: lib/modules/platform/codecommit/index.md [hunk 2]
```diff
@@ -143,3 +143,56 @@ module.exports = {
...
+    LOG_LEVEL: 'debug'
+    AWS_REGION: 'us-east-1'
+phases:
+  build:
+    on-failure: CONTINUE
+    commands:
+      - npm install -g renovate
+      - renovate
+```
+### Note:
```
Comment (HonkingGoose) [inline lib/modules/platform/codecommit/index.md:191]:
```suggestion

### Note:

```

File: lib/modules/platform/codecommit/index.md [hunk 3]
```diff
@@ -143,3 +143,56 @@ module.exports = {
...
+In order to reduce renovate BuildProject time and avoid `npm install`.
```
Comment (HonkingGoose) [inline lib/modules/platform/codecommit/index.md:192]:
```suggestion
In order to reduce Renovate BuildProject time and avoid `npm install`.
```

File: lib/modules/platform/codecommit/index.md [hunk 4]
```diff
@@ -143,3 +143,56 @@ module.exports = {
...
+
+It is recommended to install renovate on the BuildProject renovate job repository.
```
Comment (HonkingGoose) [inline lib/modules/platform/codecommit/index.md:194]:
```suggestion
We recommend you install Renovate on the BuildProject Renovate job repository.
```

File: lib/modules/platform/codecommit/index.md [hunk 5]
```diff
@@ -143,3 +143,56 @@ module.exports = {
...
+
+You can add `config.js` global config to the repository.
+
+You can add the BuildProject repository to the `RENOVATE_REPOSITORIES` and get updates on new renovate versions.
```
Comment (HonkingGoose) [inline lib/modules/platform/codecommit/index.md:198]:
```suggestion
You can add the BuildProject repository to the `RENOVATE_REPOSITORIES` variable and get updates on new Renovate versions.
```

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
Some small fixes to start us with.

**Review by rarkins — DISMISSED** [review]

File: lib/modules/platform/codecommit/index.md [hunk 6]
```diff
@@ -142,3 +142,59 @@ module.exports = {
...
+    LOG_LEVEL: 'debug'
+    AWS_REGION: 'us-east-1'
+phases:
+  build:
+    on-failure: CONTINUE
+    commands:
+      - docker run --rm -e AWS_REGION -e RENOVATE_CONFIG -e RENOVATE_PLATFORM -e RENOVATE_REPOSITORIES -e LOG_LEVEL renovate/renovate
+```
+
+### Renovate cli buildspec.yml
```
Comment (HonkingGoose) [inline lib/modules/platform/codecommit/index.md:171]:
```suggestion
### Renovate CLI `buildspec.yml` 
```

File: lib/modules/platform/codecommit/index.md [hunk 7]
```diff
@@ -142,3 +142,59 @@ module.exports = {
   ],
 };
 ```
+
+## CodeBuild examples
+
+Create a repository with `buildspec.yml`.
+This repository will be your BuildProject job repository to run Renovate on your repositories.
+
+### Renovate docker buildspec.yml
```
Comment (HonkingGoose) [inline lib/modules/platform/codecommit/index.md:151]:
```suggestion
### Renovate Docker `buildspec.yml` 
```

File: lib/modules/platform/codecommit/index.md [hunk 8]
```diff
@@ -142,3 +142,59 @@ module.exports = {
...
+    AWS_REGION: 'us-east-1'
+phases:
+  build:
+    on-failure: CONTINUE
+    commands:
+      - npm install -g renovate
+      - renovate
+```
+
+### Note:
```
Comment (HonkingGoose) [inline lib/modules/platform/codecommit/index.md:192]:
```suggestion
### Notes
```

File: lib/modules/platform/codecommit/index.md [hunk 9]
```diff
@@ -142,3 +142,59 @@ module.exports = {
...
+
+In order to reduce Renovate BuildProject time and avoid `npm install`.
```
Comment (HonkingGoose) [inline lib/modules/platform/codecommit/index.md:194]: I think we need some more explanation here.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
It feels like we're missing some more background information:

- why should users use this?
- why is it better than what they're already using
- explain benefits and drawbacks of this method

File: lib/modules/platform/codecommit/index.md (hunk 9 above)
Comment (rarkins) [reply lib/modules/platform/codecommit/index.md:194]: I prefer to merge this as it's a big improvement over what we have

**Review by rarkins — APPROVED** [review]

**Pull request merged**